lib/pact_broker/ui/views/index/show.haml

```diff
@@ -72,4 +72,21 @@
       //td, so need to append it to a div
       $(td).tooltip({container: $(td).first()});
     });
+    $('.maybe-truncate').hover(function(){
```

Comment:
Duplicated code I would like to remove.
Left it here as `show-with-tag` already has some duplicate code with `show`.

Any thought?